engine/src/main/java/org/camunda/bpm/engine/task/TaskQuery.java

```diff
@@ -391,6 +391,15 @@
    **/
   TaskQuery taskDefinitionKeyLike(String keyLike);
 
+  /**
```

Comment:
Please add meaningful JavaDoc. You can copy and adapt the documentation for taskDefinitionKeyLike.

Reply:
Copy that